server/src/test/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessorTest.java

```diff
@@ -26,6 +26,7 @@
 import java.util.Collections;
```

Comment:
use this line to change the time partition and add some test over different time partition. ```IoTDBDescriptor.getInstance().getConfig().setPartitionInterval(100);```

do not forget to set it back to original value. You can refer to ```IoTDBMultiSeriesIT.java``` for this. Thank you very much XD

Reply:
Thank you for your review! I added test cases for time partition = 100, 500 and 1000.